packages/blocks/src/divider-block/divider-block.ts

```diff
@@ -32,6 +36,13 @@ export class DividerBlockComponent extends NonShadowLitElement {
   firstUpdated() {
     this.model.propsUpdated.on(() => this.requestUpdate());
     this.model.childrenUpdated.on(() => this.requestUpdate());
+    const { page } = this.model;
+    const nextSibling = page.getNextSibling(this.model);
+    const parent = page.getParent(this.model);
+    if (!nextSibling) {
+      const id = page.addBlock('affine:paragraph', {}, parent);
+      asyncFocusRichText(page, id);
+    }
```

Comment:
Consider moving the side effects to the slash menu config instead of performing them when mounting the UI.

https://github.com/toeverything/blocksuite/blob/7c5010cdbc33a315bf434fb601b54d3c56d289f4/packages/blocks/src/components/slash-menu/config.ts#L70-L77